system_tests/storage.py

```diff
@@ -118,7 +118,6 @@ class TestStorageWriteFiles(TestStorageFiles):
 
     def test_large_file_write_from_stream(self):
         blob = self.bucket.blob('LargeFile')
-        self.assertEqual(blob._properties, {})
```

Comment:
Ditto here. This is just checking that the `Bucket.blob` factory starts out with no properties. Can be done in a unit test.